requests/test_checkbox.json

```diff
@@ -1,40 +0,0 @@
-{
```

Comment:
I think we should keep this test file as it's a nice short example, that can also be used when testing changes to the benchmarking code.